libs/langchain/langchain/chains/sql_database/query.py

```diff
@@ -13,7 +13,47 @@
     from langchain_community.utilities.sql_database import SQLDatabase
 
 
+import re
+
+
 def _strip(text: str) -> str:
```

Comment:
Currently this can be done in user code by suffixing a custom strip using LCEL composition.

I'm moving this PR into needs support.

If it gets enough up-votes, to be merged, it'll require:

1. unit tests
2. allowing to pass a custom strip function as part of the API itself (and maybe improving a default implementation together with a good explanation of the what the default implementation does / why its needed)

Reply:
The main problem is that it doesn't seem like this solution is general enough to be useful across many situations, so users will still likely need to use tackle a post-processor using LCEL composition even if this PR were to be merged